src/main/flight/mixer.c

```diff
@@ -350,6 +350,14 @@ static void applyMixToMotors(float motorMix[MAX_SUPPORTED_MOTORS], motorMixer_t
 {
     // Now add in the desired throttle, but keep in a range that doesn't clip adjusted
     // roll/pitch/yaw. This could move throttle down, but also up for those low throttle flips.
+    for (int i=0; i < mixerRuntime.motorCount; i++) {
+        if(motorOutputMixSign * motorMix[i] + throttle * activeMixer[i].throttle > 1) {
+            throttle = (1 - motorOutputMixSign * motorMix[i]) / activeMixer[i].throttle;
```

Comment:
Ah, I see.

What exactly do you mean with:
> This will also remove the cases where airmode is set off.

Do you know some method to have airmode enabled, but automatically stop the motors if the throttle is 0?
That is actually working with my fix: Motor stop at 0 throttle and but full pid authority during the flight.

Reply:
I believe the current convention is to turn on/off air mode from the radio based on throttle position.

Reply:
If you want to allow motors to be not running at zero throttle, then be very aware that this can be very dangerous and has caused injuries, because you can land, the motors are not running, but an accidental touch on the throttle and the quad can leap into the air.
It is possible to do this, if you accept the risk, by:
- setting AirMode as a Mode, on a switch, rather than the default of always on
- enabling 'Motor Stop'
- using a three-position arming switch, where the centre position is armed with AirMode off, and the full 'on' position is armed with AirMode on.
You can then fly in full 'on' position with Airmode active, but in the middle position, AirMode will be off, and when you cut throttle to zero, motors will stop.  
But remember to go to the fully off position!  
And use a two-switch method to avoid accidental arming.

Reply:
Ah ok. Nice. Thanks a lot for your help!